engine/flink/executor/src/test/scala/pl/touk/nussknacker/engine/process/helpers/ProcessTestHelpers.scala

```diff
@@ -116,3 +128,16 @@ object ProcessTestHelpersConfigCreator extends EmptyProcessConfigCreator {
   }
 
 }
+
+case object MonitorEmptySinkWithParameter extends EmptySink with Serializable {
```

Comment:
I would rather call it `SinkAccessingNodeContext`. I don't understand what is it monitor and I don't understand what parameter it has.

Reply:
Done :)